metadatastore/commands.py

```diff
@@ -671,24 +818,34 @@ def find_run_stops(run_start=None, **kwargs):
     -------
     run_stop : iterable of metadatastore.document.Document objects
     """
-    _format_time(kwargs)
-    # get the actual mongo document
     if run_start:
-        run_start = _get_mongo_document(run_start, RunStart)
-        kwargs['run_start_id'] = run_start.id
+        run_start_uid = _get_uid(run_start)
```

Comment:
`run_start` can be a uid or a document. If `run_start` does not have a uid then it is a uid